homeassistant/components/airq/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "domain": "airq",
+  "name": "air-Q",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/airq",
+  "requirements": ["aioairq==0.1.1"],
+  "ssdp": [],
+  "zeroconf": [
+    {
+      "type": "_http._tcp.local.",
+      "name": "*_air-q.local"
+    }
+  ],
```

Comment:
There's no zeroconf step in the config flow?

Reply:
Thanks for spotting it. Unfortunately, I had to retract this as our zeroconf support is not yet quite ready.